<launch>
    <node pkg="omo_r1_bringup" type="core_node.py" name="CoreNode">
        <rosparam file="$(find omo_r1_bringup)/params/core_params.yaml" command="load"/>
        <rosparam file="/root/robot_db/params/bridge_config.yaml" command="load"/>
    </node>
</launch>
